src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -90,6 +90,7 @@ void createConsumerWrongFormat(VertxTestContext context) throws InterruptedExcep
     }
 
     @Test
+    @DisabledIfEnvironmentVariable(named = "STRIMZI_USE_SYSTEM_BRIDGE", matches = "TRUE")
```

Comment:
you are disabling the test checking that `STRIMZI_USE_SYSTEM_BRIDGE` env var is set to `TRUE` but you start the local bridge if the env var is null (so it's not set). It means that if it's set to `TRUE`, `FALSE`, `Paolo`, `Renato` or whatever it doesn't start the local bridge to use the external one. We should have consistency so disabling this test if env var is set, the value seems to be not important to us or checking that the env var is `FALSE` not just null when we decide to start local bridge or not.

Reply:
Ok, I am using the external bridge only if the env var STRIMZI_USE_SYSTEM_BRIDGE is set to TRUE.
Any other value or unset env var will use internal bridge.

Reply:
the check you do when start the bridge or not is using ignoring case, while here we are using "TRUE". What happens with "true"?